CONTRIBUTING.md

```diff
@@ -202,7 +224,8 @@ message to the broker, destined for one of the topics `yggd` is subscribed to.
 To watch a topic for messages, subscribe to it with `sub`:
 
 ```
-sub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$CLIENT_ID/data/in -topic yggdrasil/$CLIENT_ID/data/out -topic yggdrasil/$CLIENT_ID/control/out
+sub -broker tcp://localhost:1883 -topic yggdrasil/$CLIENT_ID/data/in \
+    -topic yggdrasil/$CLIENT_ID/data/out -topic yggdrasil/$CLIENT_ID/control/out
```

Comment:
Since we're using `$(hostname)` for client ID in the quickstart, let's change the references here too.

```suggestion
sub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/data/in \
    -topic yggdrasil/$(hostname)/data/out -topic yggdrasil/$(hostname)/control/out
```